tests/personal_data/positions/types/test_inverse_position.py

```diff
@@ -248,7 +246,7 @@ async def test_get_bankruptcy_price_with_short(future_trader_simulator_with_defa
     position_inst.entry_price = constants.ONE_HUNDRED
     await position_inst.update(update_size=-constants.ONE_HUNDRED, mark_price=constants.ONE_HUNDRED)
     assert position_inst.get_bankruptcy_price(decimal.Decimal("20"), position_inst.side, with_mark_price=True) == \
-           decimal.Decimal("16.66666666666666666666666667")
+           decimal.Decimal("116.6666666666666666666666667")
```

Comment:
Can you please explain this change? A price change has no impact in the bankruptcy price? (this test seems to have a 20 price and the second one has a 100 price and the bankruptcy price does not change)

Reply:
Right it does not change compared the the next line because `with_mark_price=True` so we user the position mark_price and not the given price arguement.
The previous value was due to a mistake in the ternary operator in `get_bankruptcy_price` impl